spec/locales-currencies-tz.html

```diff
@@ -414,4 +414,11 @@ <h1>
       </dl>
     </emu-clause>
   </emu-clause>
+
+  <emu-clause id="sec-pattern-string-types">
+    <h1>Pattern String Types</h1>
+    <p>
+      <dfn>Pattern String</dfn> is a String value which contains zero or more substrings of the form *"{name}"*. The syntax of the abstract pattern strings is an implementation detail and is not exposed to users of ECMA-402.
```

Comment:
```suggestion
      <dfn>Pattern String</dfn> is a String value which contains zero or more substrings of the form *"{key}"*, where key can be any nonempty sequence consisting only of elements from the ASCII word characters. The syntax of the abstract pattern strings is an implementation detail and is not exposed to users of ECMA-402.
```
(trying to avoid use of "name" misleading readers into thinking that e.g. "{0}" and "{1}" are not supported)